CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 - Bugfix: Fix split focusing being broken in certain circumstances when the "Show input when it's empty" setting was disabled. (#3838, #3860)
 - Bugfix: Always refresh tab when a contained split's channel is set. (#3849)
 - Bugfix: Drop trailing whitespace from Twitch system messages. (#3888)
+- Bugfix: Fix crash caused by opening a reply thread from a usercard opened with /usercard (#3905)
```

Comment:
nit: Should this PR instead be included in the overall "Added support for Twitch's Chat Replies" changelog entry?

Reply:
Yeah, i'll move it